cmd/deck/main.go

```diff
@@ -500,13 +497,15 @@ func main() {
 		return
 	}
 
+	var server *http.Server
 	if csrfToken != nil {
 		CSRF := csrf.Protect(csrfToken, csrf.Path("/"), csrf.Secure(!o.allowInsecure))
-		logrus.WithError(http.ListenAndServe(":8080", CSRF(traceHandler(mux)))).Fatal("ListenAndServe returned.")
```

Comment:
lol sneaky

Reply:
Really, it's like it was written to deceive